apps/website/src/routes/dashboard/household/[id=ulid]/+page.svelte

```diff
@@ -110,6 +119,19 @@
   </header>
   <div class="flex gap-4">
     <main class="flex-grow">
+      <Header tag="h3">
```

Comment:
For reference, I'm not very huge on this visually speaking, but I do find that it's a better setup than what I tried before.